html-forms/<!DOCTYPE html>.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML-Forms</title>
+  </head>
+  <body>
+    <h1><strong> Coding Form! </strong></h1>
+    <label for="Email"> Email:</label>
+    <div>
+      <input type="text" id="Email1" name="Email" required />
+    </div>
+    <label for="WhyIsCodingCool">Why is Coding Cool?</label>
+    <label for="WhyIsCodingCool">Select an Option</label>
```

Comment:
This label on line 13 isn't needed. You have a label of `Why is Coding Cool?` on line 12 which is the only one you need. The `Select an Option` text is from your first option.